src/components/SearchBox/SearchBox.test.tsx

```diff
@@ -5,18 +5,23 @@ import SearchBox from "./SearchBox";
 
 describe("SearchBox ", () => {
   it("renders", () => {
-    const wrapper = shallow(<SearchBox />);
+    const wrapper = shallow(<SearchBox onSearch={jest.fn()} />);
```

Comment:
Can you add a test to check that `onSearch` is actually called when you expect it to?